### PR TITLE
Nanosuit var tweaks, renames and code cleanup [READY]

### PR DESCRIPTION
**Pull request by yoyobatty**
[Guidelines]: # (Be sure that your PR follows our guidelines, such as modularization and comment standards. You can read more about the subject here: https://github.com/HippieStation/HippieStation/blob/master/hippiestation/README.md )

[Changelogs]: # (Your PR should contain a detailed changelog of notable changes, titled and categorized appropriately. An example changelog has been provided below for you to edit. If you need additional help, read https://github.com/tgstation/tgstation/wiki/Changelogs )

:cl: YoYoBatty
add: Added atmospheric sensors to nanosuit stat tracks
add: Added more damage voice feedback to nanosuit
tweak: Improved nanosuit logging
code: Improved nanosuit variable names and made code a little more readable.
balance: Nanosuit cloak use reduced to 1.2 from 1.3 and speed use reduced to 1.6 from 1.8
/:cl:

[why]: 

Let me know what you think of this, I'm trying to make the nanosuit a tad less boring and offer it some more robust capabilities in unarmed combat.

### CODE REVIEW

**Comment by yoyobatty** [comment]
Please note that much of this pr is a work in progress so the code I have is subject to change and open to ideas and suggestions

**Comment by ghost** [comment]
according to your description this sounds like a worthwhile addition to the game, I particularly like the 30% chance of 7s knockdown.

+1

**Comment by yoyobatty** [comment]
I'm probably going to change it though, community isn't a huge fan of rng right now anyway.

**Comment by DaydreamPuppeh** [comment]
I think nanosuit needs to be nerfed before it can receive a buff.

**Comment by ghost** [comment]
It has come to my understanding that the current nanosuit is considered ''OP'', I do not feel that is the case but if that is indeed the case and most people agree I will have to retract my +1 for the benefit of the community as a whole.

Greetings, Fluxcapacitor1337
Headmin

**Comment by yoyobatty** [comment]
This will be updated soon. Just out.

**Comment by Carbonhell** [comment]
add [READY] before your title to mark it as ready for review when done btw.

File: hippiestation/code/modules/clothing/spacesuits/nanosuit.dm [hunk 1]
```diff
@@ -764,7 +781,7 @@
 		D.visible_message("<span class='danger'>[A] attempted to disarm [D]!</span>", \
 							"<span class='userdanger'>[A] attempted to disarm [D]!</span>")
 		playsound(D, 'sound/weapons/punchmiss.ogg', 25, 1, -1)
-	log_combat(A, D, "disarmed with krav maga", "[I ? " removing \the [I]" : ""]")
```
Comment (Carbonhell) [inline hippiestation/code/modules/clothing/spacesuits/nanosuit.dm:767]: how did we miss this lmfao

**Review by Carbonhell — APPROVED** [review]

**Comment by Carbonhell** [comment]
ready to go?

**Comment by yoyobatty** [comment]
Yes sir!

**Pull request merged**